utils/serialise.js

```diff
@@ -1,12 +1,13 @@
 module.exports = serialise
 
-function serialise (type, attributes, id) {
+function serialise (type, attributes, id, createdAt) {
   if (!type || !attributes) {
     throw new Error('Serialisation must include a type and some attributes.')
   }
   var data = {
     type: type,
-    attributes: attributes
+    attributes: attributes,
+    createdAt: createdAt
   }
```

Comment:
After going through the possibilities, I think it makes sense to bring in lolex into `unit/sign-up-test.js`. I think this should be the last change I need to make for this PR. Please let me know if I'm missing something or if I can better refactor a part of the submission. Thanks :)